Core/DISET/private/BaseClient.py

```diff
@@ -205,7 +205,7 @@ def __findServiceURL( self ):
       
     if len( urls ) == len( self.__bannedUrls ):
       self.__bannedUrls = []  # retry all urls
-      self.__retryDelay = 5 / len ( urls )  # we run only one service! In that case we increase the retry delay.
+      self.__retryDelay = 5 / len ( urls ) if len( urls ) > 1 else 1  # we run only one service! In that case we increase the retry delay.
```

Comment:
Why is the shortest delay in case of 1 service ? Aren't you contradicting yourself when you say the more the number of services the less delay ?